pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Description
+{ This PR adds ... }
+
+{ This PR fixes ... }
+
+{ This PR removes ... }
+
+## Due Diligence Checklist
+- [ ] Tests
+- [ ] Documentation
+- [ ] Consolidated commits
+- [ ] Relevant labels assigned
+- [ ] Changelog(s) updated
+
+## Steps to Test
+- `yarn test`
+- { Specific instructions }
+
+## Deploy Notes
+- { `yarn install` }
+- { `yarn migrate` }
+- { Populate environment variables ... }
+- { Set up ... }
+
+## Related Issues
```

Comment:
This is almost always in the PR title or description. Remove this section and add it there.

Reply:
I disagree; having a separate section allows for PRs to be related to multiple issues, which is sometimes the case. I also find including issues in the title to lead to hard-to-read titles that are less likely to effectively communicate their intent.